test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1988,6 +2045,15 @@ def setup_rnn_tests():
                               (unittest.TestCase,),
                               dict(TestCaffe2Backend.__dict__, embed_params=True))
 
+# opset 10 tests
+TestCaffe2Backend_opset10 = type(str("TestCaffe2Backend_opset10"),
```

Comment:
Also I am wondering if we can renamed TestCaffe2Backend and TestCaffe2BackendEmbed to TestCaffe2Backend_opset9 and TestCaffe2BackendEmbed_opset10

If so, let's do it in the following PRs.